src/transformers/models/bart/modeling_flax_bart.py

```diff
@@ -341,6 +341,67 @@ def __call__(
         return outputs
 
 
+class FlaxBartEncoderLayerCollection(nn.Module):
```

Comment:
This is needed to imitate the exact same weight structure as in PyTorch.

In PyTorch we always have things like `bart.encoder.layers.1` .... => this means that we need to create a class in Flax which represents `layers` and give each layer the weights name `1`, `2`, ...